packages/web3-eth-accounts/src/index.js

```diff
@@ -42,6 +39,49 @@ var isNot = function(value) {
     return (_.isUndefined(value) || _.isNull(value));
 };
 
+var flattenBytes = function(a) {
```

Comment:
@GregTheGreek Yes, I'm getting stuck with the `Account` package. There don't seem to be direct mappings in `ethereumjs-utils` for most of the package. I've added the functions from `eth-lib` inline, within the file as helpers (has missing references right now) and planning to cleanup each function to use internal utils instead. If they don't exist already, will add them to `web3-utils` (eg, Bytes.flatten, Bytes.slice etc).  Does that sound like the right approach?

Reply:
I think a lot of the functions you want might be [here](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/util/src/signature.ts) to remove code in `hashMessage`?

Also for the `recoverTransaction` you're going to run into some problems becasue of berlin support, maybe @luu-alex can make a gist for you showing how it should be supported?